### PR TITLE
using yui-ibc-solidity v0.3.3

### PR DESCRIPTION
**Pull request by 3100**

### CODE REVIEW

File: samples/minitoken-besu-ethereum/package.json [hunk 1]
```diff
@@ -19,11 +19,11 @@
     }
   ],
   "devDependencies": {
-    "@truffle/hdwallet-provider": "^1.2.1",
-    "truffle": "5.4.16",
+    "@truffle/hdwallet-provider": "^2.0.15",
+    "truffle": "5.6.6",
     "truffle-contract-size": "^2.0.1"
```
Comment (bluele) [inline samples/minitoken-besu-ethereum/package.json:24]: @3100 Do we need `truffle-contract-size` in this tutorial? (I understand this is out of scope)

Comment (3100) [reply samples/minitoken-besu-ethereum/package.json:24]: Indeed, it's not being used. I will remove it.

**Review by bluele — APPROVED** [review]
